### PR TITLE
About me section layout desktop only

### PR DESCRIPTION
**Pull request by Alekslov**

### CODE REVIEW

**Review by vaidas33 — DISMISSED** [review]

**Review by Vaidmine — DISMISSED** [review]
all oK except button, but it will be styled separately...

File: css/home-page.css [hunk 1]
```diff
@@ -27,6 +27,9 @@ CUSTOM HOME PAGE STYLES GOES BELOW THIS COMMENT
 /* header */
 /* hero-home */
 /* about-me */
+.about {
+    box-shadow: none;
+}
```
Comment (anavisnia) [inline css/home-page.css]: nereikalingi

**Review by anavisnia — APPROVED** [review]
LG

**Review by vaidas33 — APPROVED** [review]
Good

**Pull request merged**